vane/tests_tools.py

```diff
@@ -269,6 +269,8 @@ def login_duts(test_parameters, test_duts):
         login_ptr["username"] = dut["username"]
         login_ptr["password"] = dut["password"]
         login_ptr["role"] = dut["role"]
+        if "neighbors" not in dut:
+            dut["neighbors"] = ""
         login_ptr["neighbors"] = dut["neighbors"]
```

Comment:
As in the other PR, use dictionary.get() instead seems more pythonic
```suggestion
        login_ptr["neighbors"] = dut.get("neighbors", "")
```